.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,81 @@
+name: CodeCoverage
+
+on:
+  schedule:
+    # 2AM every Tuesday/Thursday
+    - cron:  '0 2 * * 2,4'
+jobs:
+    Build:
+        strategy:
+            matrix:
```

Comment:
Can also be written as:

```
matrix:
  os: ubuntu-latest

runs-on: ${{matrix.os}}
```

Reply:
I’ve just done the minimum amount of changes to make this work. I.E copy/paste/delete